src/github.ts

```diff
@@ -106,41 +106,48 @@ export const createTags = async (
   )
 }
 
+const generateReleaseNotes = async (
```

Comment:
Please rename this to, e.g. "generateReleaseNotesFromFile", so we can later easily add a "generateReleaseNotesFromFolderStructure", like we do it for integreat.

Reply:
I don't get the naming. We generate it not from a file but from the api endpoint of github

Reply:
by file i ment the "Changelog.md", but "generateReleaseNotesFromGithubEndpoint" is also fine